glowing_waffles/differential_photometry/one_vsx_mag.py

```diff
@@ -0,0 +1,35 @@
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+def calcVmag(var_stars, star_data, comp_stars):
+    var_coords = var_stars['coords']
+    star_data_coords = SkyCoord(ra=star_data['RA'], dec=star_data['Dec'])
+    v_index, v_d2d, _ = var_coords.match_to_catalog_sky(star_data_coords)
+    rcomps = comp_stars[comp_stars['band']=='Rc']
```

Comment:
```suggestion
    rcomps = comp_stars[comp_stars['band'] == 'Rc']
```
(another really important change 🙄 )